packages/ui/src/components/Confirm/index.tsx

```diff
@@ -0,0 +1,40 @@
+import Portal from '../Portal';
+import Styled from './Confirm.styles';
+
+interface ConfirmProps {
+  children: React.ReactNode;
+  cancelText?: string;
+  confirmText?: string;
+  onCancel?: () => void;
+  onConfirm?: () => void;
+}
+
+const Confirm = ({ children, cancelText, confirmText, onCancel, onConfirm }: ConfirmProps) => {
+  const handleCancel = () => {
+    onCancel?.();
+  };
+
+  const handleConfirm = () => {
+    onConfirm?.();
+  };
+
+  return (
+    <Portal>
+      <Styled.DimmedArea>
+        <Styled.Confirm>
+          <Styled.ConfirmMessage>{children}</Styled.ConfirmMessage>
+          <Styled.ConfirmButtonContainer>
+            <Styled.CancelButton type="button" onClick={handleCancel}>
+              {cancelText ?? '취소'}
+            </Styled.CancelButton>
+            <Styled.ConfirmButton type="button" onClick={handleConfirm}>
+              {confirmText ?? '확인'}
+            </Styled.ConfirmButton>
```

Comment:
따로 핸들러 분리한 이유가 있을까?!

```suggestion
            <Styled.CancelButton type="button" onClick={onCancel}>
              {cancelText ?? '취소'}
            </Styled.CancelButton>
            <Styled.ConfirmButton type="button" onClick={onConfirm}>
              {confirmText ?? '확인'}
            </Styled.ConfirmButton>
```

Reply:
저기에 뭔가 더 넣게 될 줄 알고 저렇게 만들었는데, 결국 안 들어가게 되어서 그대로 남게 됐어... ㅋㅋㅋ 수정해둘게!